cpp/src/Ice/UserException.cpp

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#include "Ice/UserException.h"
+#include "Ice/InputStream.h"
+#include "Ice/OutputStream.h"
+
+using namespace std;
+using namespace Ice;
+using namespace IceInternal;
```

Comment:
Seems `IceInternal` is not required here.

Reply:
Removed